src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1569,18 +1569,29 @@ core::Error runGrepOperation(const GrepOptions& grepOptions, const ReplaceOption
    return Success();
 }
 
+std::string escapeCharacters(const std::string& query)
+{
+   std::string specialChars("([.|*?+(){}^$\\[\\]])");
```

Comment:
IIRC we also considered whether we could use the PCRE escapes `\Q` and `\E` here (https://www.pcre.org/original/doc/html/pcresyntax.html#SEC2) but that didn't work out. Is that right?

Reply:
Yeah, I believe that's right -- I can double check but that's my recall as well. I haven't gotten back to look at this since we pulled it out of SW so will test this out again before merging